tools/df/explain_mrt/explain.py

```diff
@@ -0,0 +1,1065 @@
+"""
```

Comment:
Work in progress on MRT Dump analysis for errors

This code is not distributed in the final debian